cptv-reprocess.py

```diff
@@ -23,14 +24,15 @@ def reprocess(args):
         "processingState": {"$ne": "FINISHED"},
     }
 
-    if args.lower_id:
-        id_where = where.get("id", {})
-        id_where["$gte"] = args.lower_id
-        where["id"] = id_where
-    if args.upper_id:
-        id_where = where.get("id", {})
-        id_where["$lte"] = args.upper_id
-        where["id"] = id_where
+    if len(args.recording_id) == 2:
+        if args.recording_id[0]:
+            id_where = where.get("id", {})
+            id_where["$gte"] = int(args.recording_id[0])
+            where["id"] = id_where
```

Comment:
these three lines could be reduced to: 

```python
where.setdefault("id", {})["$gte"] = int(args.recording_id[0])
```

or if you want to spread it out a bit:
```python
id_where = where.setdefault("id", {})
id_where["$gte"] = int(args.recording_id[0])
```